src/SDK/Common/Time/ClockFactory.php

```diff
@@ -4,27 +4,16 @@
 
 namespace OpenTelemetry\SDK\Common\Time;
 
-final class ClockFactory implements ClockFactoryInterface
-{
-    private static ?ClockInterface $default = null;
-
-    public static function create(): self
-    {
-        return new self();
-    }
-
-    public function build(): ClockInterface
-    {
-        return new SystemClock();
-    }
+use OpenTelemetry\API\Common\Time\ClockInterface;
 
+/**
+ * @deprecated Use OpenTelemetry\API\Common\Time\ClockFactory
+ * @codeCoverageIgnore
+ */
+class ClockFactory
```

Comment:
:thinking: Should the old interface should remain too and also marked as deprecated?
```suggestion
class ClockFactory implements ClockFactoryInterface
```

then maybe something like:
```suggestion
interface ClockFactoryInterface extends \OpenTelemetry\API\Common\Time\ClockFactoryInterface
```

Reply:
Retained old SDK interfaces and Util class, deprecated.

Reply:
I'm prepared to gamble on nobody having done that, and wear the BC break. There'll be a beta of the next version.

Reply:
I think that in the unlikely event that somebody has implemented their own clock factory, then it can get picked up when we release a beta. I did try to keep it around, but deptrac was unhappy and I didn't think it was worth the effort.